docs/private-app-manage.md

```diff
@@ -0,0 +1,22 @@
+## How to manage Private Apps 
+
+### List Private Apps
+
+Via the Private App management page, click on the Manage Private Apps button (Fig 1) to see a list of enabled and disabled Private Apps . 
+
+![image](https://github.com/HDRUK/gateway-2-integrations-testing/assets/69473770/f363f047-6d33-4030-bda1-3fbedd612f7c)
+_Fig 1: Manage Private Apps_
+
+Each App is listed with its title, creation date, App ID, and a brief description. The list (Fig 2) can be filtered by App status. 
+
+![image](https://github.com/HDRUK/gateway-2-integrations-testing/assets/69473770/75c7784e-35f4-4a40-b58a-0fb8007c0a0b)
+_Fig 2: List of Private Apps_
+
+
+### Update a Private App 
+
+Select the App you wish to modify, and then update the App information and permissions settings. Click the Save button to store the updated information on the Gateway. A pop-up window will confirm the successful App update . 
+
+The App’s ClientID will not automatically regenerate when permissions or information is changed. If you are making significant changes to the permissions of an App we recommend either disabling or deleting the old Private App and creating a new one. 
+
+Please note that deleting an App is an irreversible action. 
```

Comment:
Should we make this section bold or somehow make it stand out? Seems like quite important info!

Reply:
In next merge :)